### DISCUSSION TITLE
Difficulties with JetBrains IDE autocomplete popups

### DISCUSSION THREAD

**Discussion by russjr08** [hyprwm/Hyprland#3864] [Q&A]
Hi there! I wasn't sure if this was necessarily a bug, so figured I would open a Q&A discussion instead - apologies if I'm wrong about that.

I'm having an issue with a certain type of popup window that the various JetBrains IDEs create whenever an autocompletion suggestion is made - specifically, the contents of the window seem to "shrink", but there's still a visual blur where the window *should* be. It's a bit hard to explain, so I've captured a video to demonstrate:

https://github.com/hyprwm/Hyprland/assets/773179/09ead8fd-e0e5-48e3-8332-61604b324d40

I've done some searching around to try to tackle this issue, and came across [this issue](https://github.com/hyprwm/Hyprland/issues/2117) but none of the suggested solutions seemed to work - but that's not too surprising given that the recommended solutions were for a different type of window (that I don't have any issues with on my system).

I did attempt to try out pretty much all of the documented window rules just to see if I could effectively get Hyprland to "ignore" the popup with a quick rudimentary windowrule like `windowrulev2 = noblur,title:^(win.*)$` (of course, replacing `noblur` with all of the available rule options). I can see that it does match, as if I set the rule to something like `tile` it certainly forces the popup into tiling mode. 

So I'm just wondering if perhaps anyone else has run into the same issue and might know of a solution, or perhaps have run into something similar and what they used to resolve the problem.

**Comment by gslandtreter** [comment]
I'm running into the same issue. It started happening recently. I believe v0.31.0 onwards. Haven't been able to find a workaround yet.

**Comment by AWerbrouck** [comment]
@russjr08 Have you found a solution for this? 
it seems to be the same issue as here #2579

**Reply by russjr08** [reply]
@AWerbrouck Unfortunately I have not, based on the discussion linked in the issue from that issue, looks like we're SOL since the issue was closed as not planned. 

This will be a deal-breaker for me since I use JBR's IDEs every day for work. 

Given the response and status of said issue, I'll just close this QA out.

**Comment by gslandtreter** [comment]
Hi @russjr08 and @AWerbrouck, FYI, I didn't have time to check the diffs and what's the root cause, but rolling Hyprland back to v0.28.0 has solved the problem. Not a great workaround, but at least something we can do for now.

https://github.com/hyprwm/Hyprland/releases/tag/v0.28.0

**Reply by DJsushi** [reply]
Hyprland v0.30.0 still hasn't got the issue. I think it happened somewhere v0.31+

**Comment by yzld2002** [comment]
Seems v0.34.0 has solved this issue.
